core/iwasm/aot/aot_loader.c

```diff
@@ -3749,6 +3814,11 @@ load_from_sections(AOTModule *module, AOTSection *sections,
                     return false;
                 break;
             case AOT_SECTION_TYPE_TEXT:
+#if defined(BUILD_TARGET_AARCH64)
```

Comment:
comment for why need this #ifdef is added
if that [unconditional try to merge](https://github.com/bytecodealliance/wasm-micro-runtime/pull/3681#discussion_r1704794923) will apply, then this will be changed.